spec/requests/comment_spec.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Comments', type: :request do
+  describe 'NEW post comment' do
+    let!(:user) { create(:user) }
+    let!(:newpost) { create(:post) }
+    let(:content) { Faker::Lorem.paragraph }
+    let(:comment_params) do
+      { comment: { content:, commentable_type: newpost.class.name, commentable_id: newpost.id } }
```

Comment:
You can use the method `attributes_for`, and so you don't need to define the variable `content` as you would use the factory for that